code/modules/jobs/job_types/nobility/magician.dm

```diff
@@ -56,8 +55,11 @@
 	backpack_contents = list(/obj/item/scrying = 1, /obj/item/chalk = 1,/obj/item/reagent_containers/glass/bottle/killersice = 1, /obj/item/book/granter/spellbook/master = 1, /obj/item/weapon/knife/dagger/silver/arcyne = 1, /obj/item/storage/keyring/mage = 1,)
 
 	H.adjust_skillrank(/datum/skill/misc/reading, 6, TRUE)
-	H.adjust_skillrank(/datum/skill/magic/arcane, pick(6,5), TRUE)
+	H.adjust_skillrank(/datum/skill/magic/arcane, pick(5,6), TRUE)
 	H.adjust_skillrank(/datum/skill/misc/riding, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE)
```

Comment:
they are old mages , why would they be more athletic than a towner

Reply:
court magician is a member of the old party, they were a proper adventurer, and they need athletics to cast their spells without keeling over from exhausting

Reply:
>that was BEFORE they are old now
which is represented in their stats, not skills
skills are skills, knowledge and technique one has acquired